import-express.html

```diff
@@ -40,11 +40,18 @@ <h2>Import - Express</h2>
                 <div class="section-cols">
                     <div class="section-col">
                         <form class="import-form">
-                            <sp-field-label for="import-url" required>URL</sp-field-label>
-                            <sp-textfield class="option-field" id="import-url" type="url">
-                                <sp-help-text slot="help-text">https://www.example.com</sp-help-text>
-                                <sp-help-text slot="negative-help-text">Please enter a valid URL.</sp-help-text>
-                            </sp-textfield>
+                            <div id="express-url-inputs">
+                                <div>
+                                    <sp-field-label for="import-url" required>URL</sp-field-label>
+                                    <sp-textfield class="option-field" id="import-url" type="url">
+                                        <sp-help-text slot="help-text">https://www.example.com</sp-help-text>
+                                        <sp-help-text slot="negative-help-text">Please enter a valid URL.</sp-help-text>
+                                    </sp-textfield>
+                                </div>
+                                <sp-button id="express-open-url-button" title="Open URL in new tab">
```

Comment:
Should this be an `icon-only` button? Could `primary` or `secondary` variant be added too?